seqr/views/apis/dataset_api.py

```diff
@@ -116,15 +86,25 @@ def add_variants_dataset_handler(request, project_guid):
         request.user, {'analysis_status': Family.ANALYSIS_STATUS_ANALYSIS_IN_PROGRESS}, guid__in=family_guids_to_update)
 
     if project_has_analyst_access(project):
+        updated_individuals = {sample.individual_id for sample in updated_samples}
+        previous_loaded_individuals = {
+            sample.individual_id for sample in Sample.objects.filter(
+                individual__in=updated_individuals, sample_type=sample_type, dataset_type=dataset_type,
+            ).exclude(elasticsearch_index=elasticsearch_index)}
+        previous_loaded_individuals.update(matched_individual_ids)
+        new_sample_ids = [
+            sample.sample_id for sample in updated_samples if sample.individual_id not in previous_loaded_individuals]
         safe_post_to_slack(
             SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL,
-            """{num_sample} new samples are loaded in {base_url}project/{guid}/project_page
+            """{num_sample} new {sample_type}{dataset_type} samples are loaded in {base_url}project/{guid}/project_page
```

Comment:
Need a space between `{sample_type}` and `{dataset_type}`?

Reply:
no because we do not  show dataset type if its its a a standard variant, if you look at how `dataset_type` is passed to the format you will see that it handles conditionally adding the space